yarn-project/foundation/src/log/pino-logger.ts

```diff
@@ -107,9 +109,10 @@ export const levels = {
 // since pino will load this transport separately on a worker thread, to minimize disruption to the main loop.
 const otlpEndpoint = process.env.OTEL_EXPORTER_OTLP_LOGS_ENDPOINT;
 const otelOpts = { levels };
-const otelTransport: pino.TransportSingleOptions = {
+const otelTransport: pino.TransportTargetOptions = {
   target: '@aztec/telemetry-client/otel-pino-stream',
   options: otelOpts,
+  level: 'trace',
```

Comment:
Do the logs get filtered elsewhere or is this transport always going to publish >trace logs? (sorry, not familiar with the new pino logging backend yet)

Reply:
Yeah, they get filtered by the top-level `pino` instance that wraps the transport.

Reply:
Note that we could set this level to `logLevel` as well, but then when we create a child logger with a different level, the logs wouldn't make it past the transport.